### DISCUSSION TITLE
Should togglePlayBack action(?) be able to start playback when there is no active song playing by MPD?

### DISCUSSION THREAD

**Discussion by vendion** [aditya-K2/gomp#53] [General]
So far `gomp` is coming along nicely as a modern TUI client for `MPD` but as it stands, I cannot use only gomp due to one major annoyance. If MPD is in a stopped state and has no songs in the play queue, I can use gomp to go through my library and add songs to the queue no problem. What I can't do though is to use `togglePlayBack` (default keymap `p`) to start playback. To get MPD to actually start playing, I have to use a different client (`mcp`, `ncmpcpp`, etc) and then I can manage playback with gomp.

Is this the expected behavior? If so I can't find another command in the [key_map](https://github.com/aditya-K2/gomp/blob/master/config/kmap.go#L62) map that sounds like it will start playback from a stopped state.

**Comment by akurdunkar** [comment]
As of right now, this is expected behavior. But I have sometimes felt the need for `toggle_playback` to start the playback if the playback is stopped. I am leaning toward making the previously mentioned behavior the default.

**Comment by akurdunkar** [comment]
Fixed in #54
